data/operators/leisure/park.json

```diff
@@ -1395,6 +1395,25 @@
         "operator:wikidata": "Q81824"
       }
     },
+    {
+      "displayName": "City of Houston",
+      "id": "houstonparksandrecreationdepartment-8bfd34",
+      "locationSet": {
+        "include": ["us-tx.geojson"]
+      },
+      "tags": {
+        "access": "public",
```

Comment:
According to the wiki, `access=public` has been deprecated (see https://wiki.openstreetmap.org/wiki/Tag:access=public)

Reply:
So should we use `access=yes` instead to designate that the park is public?